README.md

```diff
@@ -4,128 +4,141 @@
+|                 One-Time Quick Reviews                   |       Need to view the delta between one or more runs    |
+|                Decentralized Deployment                  |   Need to view subsets of the 800-53 control alignment   |
+|                    Minimal A&A Time                      | Need to produce more complex reports in multiple formats |
+
 
-## General Deployment
 
-### Running Heimdall Lite
+## Getting Started / Installation
 
-Heimdall Lite is a standard VueJS app so for help with a local deployment, please see: <https://cli.vuejs.org/guide/deployment.html#general-guidelines>
+### Heimdall Lite
 
-Heimdall Lite is published to the <npmjs.org> MITRE site at <https://www.npmjs.com/package/heimdall-lite>.
+Heimdall Lite is published to npmjs.org and is available [here](https://www.npmjs.com/package/heimdall-lite).
 
 #### Running via npm
 
-To run Heimdall Lite locally, just use the `npm` built-in utility `npx`:
+To run Heimdall Lite locally, just use the `npm` built-in utility `npx`: 
 
-`npx heimdall-lite`
+```bash
+npx heimdall-lite
+```
 
-If you use this tool often and want to have it installed locally, use the following command:
+If you use this tool often and want to have it installed locally, use the following command: 
 
-`npm install -g heimdall-lite`
+```bash
+npm install -g heimdall-lite
+```
 
 Then, any subsequent `npx heimdall-lite` will use the local version and load much more quickly.
 
 #### Running via Docker
 
 It is also possible to run heimdall-lite using Docker, using the following command:
 
-`docker run -d -p 8080:80 mitre/heimdall-lite:release-latest`
+```bash
+docker run -d -p 8080:80 mitre/heimdall-lite:release-latest
+```
 
-You can then access heimdall-lite via: localhost:8080
+You can then access heimdall-lite at [`http://localhost:8080`](http://localhost:8080).
 
 If you would prefer to run the bleeding edge version of heimdall-lite, replace `mitre/heimdall-lite:release-latest` with `mitre/heimdall-lite:latest`.
 
-### Running Heimdall Server
+---
+
+### Heimdall Server - Docker
 
 Given that Heimdall requires at least a database service, we use Docker and Docker Compose to provide a simple deployment experience.
 
 #### Setup Docker Container (Clean Install)
 
 1. Install Docker
-2. Download heimdall by running `git clone https://github.com/mitre/heimdall2.git`
+
+2. Download and extract the most recent Heimdall release from our [releases page](https://github.com/mitre/heimdall2/releases).
+
 3. Navigate to the base folder where `docker-compose.yml` is located
-4. Run the following commands in a terminal window from the heimdall source directory:
-   1. `./setup-docker-secrets.sh`
-   2. `docker-compose up -d`
-6. Navigate to `http://127.0.0.1:3000`
+
+4. Run the following commands in a terminal window from the Heimdall source directory:
+   1. ```bash
+      ./setup-docker-secrets.sh
+      ```
```

Comment:
Triple backticks display rather poorly when in a single item list:

<img width="490" alt="Screen Shot 2021-01-12 at 1 52 35 PM" src="https://user-images.githubusercontent.com/2308869/104359375-9472f700-54dd-11eb-9bce-5f4670a6c0ef.png">

It really only works well on code blocks with 2 or more lines in them.